internal/common/rules/ExampleMatrixRegistration.go

```diff
@@ -0,0 +1,30 @@
+package rules
+
+import (
+	"gonum.org/v1/gonum/mat"
+)
+
+func RegisterCoolRool() {
```

Comment:
let's follow godoc conventions?
https://blog.golang.org/godoc

Reply:
Added in latest commit